var/spack/repos/builtin/packages/precice/package.py

```diff
@@ -153,9 +153,7 @@ def variant_bool(feature, on="ON", off="OFF"):
         if "+python" in spec:
             python_library = spec["python"].libs[0]
             python_include = spec["python"].headers.directories[0]
-            numpy_include = join_path(
-                spec["py-numpy"].prefix, spec["python"].package.platlib, "numpy", "core", "include"
-            )
+            numpy_include = spec["py-numpy"].headers.directories[0]
```

Comment:
When I try this branch out with `spack python`, I get the wrong directory. Is there something I'm missing here? I assumed that the `HeaderList` could come out in arbitrary order (although it looks alphabetical below), which is why I was thinking that breaking out a specific method to expose `numpy/core/include` might make sense:
```bash
; spack python                    
Spack version 0.19.0.dev0
Python 3.10.7, Linux x86_64
>>> from spack.spec import Spec
>>> n = Spec('py-numpy').concretized()
>>> n.package.headers.directories
['/home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-12.1.1/py-numpy-1.23.3-n4htesvvw6nvg5js5ogmoph7l2csm5kr/lib/python3.9/site-packages/numpy/f2py/src', '/home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-12.1.1/py-numpy-1.23.3-n4htesvvw6nvg5js5ogmoph7l2csm5kr/lib/python3.9/site-packages/numpy/core/include', '/home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-12.1.1/py-numpy-1.23.3-n4htesvvw6nvg5js5ogmoph7l2csm5kr/lib/python3.9/site-packages/numpy/f2py/tests/src/module_data']
>>> n.package.headers.directories[0]
'/home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-12.1.1/py-numpy-1.23.3-n4htesvvw6nvg5js5ogmoph7l2csm5kr/lib/python3.9/site-packages/numpy/f2py/src'
```

Reply:
Decided to just undo this change and leave the numpy stuff to someone else.